dom.bs

```diff
@@ -3320,6 +3320,8 @@ dictionary MutationObserverInit {
   boolean attributeOldValue;
   boolean characterDataOldValue;
   sequence&lt;DOMString> attributeFilter;
+  sequence&lt;DOMString> elementFilterByAttribute;
+  sequence&lt;DOMString> elementLocalNameFilter;
```

Comment:
Shall we name this `elementFilter` to match `attributeFilter`?

Reply:
Also, @smaug---- suggested that maybe all filters should end with `Filter`, which seems reasonable. That would suggest `elementByAttributeFilter` and `elementFilter` to me.

Reply:
That sounds like a reasonable suggestion.